WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -133,13 +133,9 @@ public CoinJoinTabViewModel(WalletService walletService)
 
 		private WalletService WalletService { get; }
 
-		private CompositeDisposable Disposables { get; set; }
-
-		public override void OnOpen()
+		public override void OnOpen(CompositeDisposable disposables)
```

Comment:
This is confusing for me. When to call `base.OnOpen()` or do I have to call it? Is this called before or after the Open? How about putting the things in `base.OnOpen()` into `IDockableViewModel.OnOpen()` - then rename `OnOpen` to something like `OnOpening(CompositeDisposable disposables)`.

Reply:
You never call OnOpen yourself, the underlying framework calls it for you.

You basically override it, if you need to do something when your tab opens.

Reply:
As explained in previous, you can no longer see base.OnOpen() now, its hidden because of the explicit interface implementation. So no confusion.

Reply:
Unable to have consistent naming because OnClose comes from interface provided by Dock, so we'd have to get another release of that to rename.

Reply:
Added comments to document those methods clarifying this.